webplugin/controller.js

```diff
@@ -1,32 +1,36 @@
 const path = require("path");
 const fs = require('fs');
+const util = require('util');
 const config = require("./../server/config/config-env");
+const {pluginVersionData} = require("./pluginOptimizer.js");
 
-exports.getPlugin = (req, res) => {
-  //var MCK_CONTEXTPATH =req.protocol+"://"+req.headers.host;
+exports.getPlugin = async (req, res) => {
+  const MCK_PLUGIN_VERSION = req.params.version;
+  if (!MCK_PLUGIN_VERSION) {
+    res.send("err while getting plugin...");
+    return console.log("unable to get plugin version");
+  }
+  var data = Object.keys(pluginVersionData).length ? pluginVersionData[MCK_PLUGIN_VERSION] : await generatePluginFile(req, res);
+  res.setHeader('content-type', 'application/javascript');
```

Comment:
@ab14bhardwaj can this case impact anything on any browser? standard is "Content-Type". 
https://en.wikipedia.org/wiki/List_of_HTTP_header_fields
headers always starts with Capital letter and have - as splitter.

Reply:
@devashishmamgain Changes done.